ReactAndroid/build.gradle

```diff
@@ -4,6 +4,7 @@ apply plugin: 'com.android.library'
 apply plugin: 'maven'
 
 apply plugin: 'de.undercouch.download'
+apply plugin: 'com.neenbedankt.android-apt'
```

Comment:
android-apt is now deprecated. with the gradle plugin 2.2+ you should use `annotationProcessor` instead